packages/libllvm/build.sh

```diff
@@ -119,16 +117,3 @@ termux_step_post_make_install() {
 		ln -f -s clang-${TERMUX_PKG_VERSION:0:1} $tool
 	done
 }
-
-termux_step_post_massage() {
-	sed $TERMUX_PKG_BUILDER_DIR/llvm-config.in \
-		-e "s|@TERMUX_PKG_VERSION@|$TERMUX_PKG_VERSION|g" \
-		-e "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" \
-		-e "s|@TERMUX_PKG_SRCDIR@|$TERMUX_PKG_SRCDIR|g" \
-		-e "s|@LLVM_TARGET_ARCH@|$LLVM_TARGET_ARCH|g" \
-		-e "s|@LLVM_DEFAULT_TARGET_TRIPLE@|$LLVM_DEFAULT_TARGET_TRIPLE|g" \
-		-e "s|@TERMUX_ARCH@|$TERMUX_ARCH|g" > $TERMUX_PREFIX/bin/llvm-config
-	chmod 755 $TERMUX_PREFIX/bin/llvm-config
```

Comment:
This was quite weird. First off all, this script was never used, from the very beginning in #917, as you have to place it in `$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX`. Instead, there's already a `llvm-config` executable, at least in the LLVM 9 package, and it seems to work fine.